includes/Shopping/Shopify_Query.php

```diff
@@ -151,17 +151,22 @@ protected function execute_query( string $query ) {
 	 * @since 1.21.0
 	 *
 	 * @param string $search_term Search term to filter products by.
+	 * @param string $after   After node.
+	 * @param int    $per_page   Limit query.
 	 * @param string $orderby Sort collection by product attribute.
 	 * @param string $order Order sort attribute ascending or descending.
 	 * @return string The assembled GraphQL query.
 	 */
-	protected function get_products_query( string $search_term, string $orderby, string $order ): string {
+	protected function get_products_query( string $search_term, string $after, int $per_page, string $orderby, string $order ): string {
 		$search_string = empty( $search_term ) ? '*' : '*' . $search_term . '*';
 		$sortkey       = 'date' === $orderby ? 'CREATED_AT' : strtoupper( $orderby );
 		$reverse       = 'asc' === $order ? 'false' : 'true';
+		$after         = empty( $after ) ? '' : $after;
+		$after_query   = empty( $after ) ? '' : " after: \"$after\",";
+
 		return <<<QUERY
 {
-  products(first: 100, sortKey: $sortkey, reverse: $reverse, query: "title:$search_string") {
+  products(first: $per_page, $after_query sortKey: $sortkey, reverse: $reverse, query: "title:$search_string") {
```

Comment:
Adding `$after_query` like this is a bit error prone, hard to read and not in line with how the other args are added.

This should work as well:

```suggestion
  products(first: $per_page, after: $after, sortKey: $sortkey, reverse: $reverse, query: "title:$search_string") {
```

And above:

```php
$after         = empty( $after ) ? null : $after;
```

Reply:
No, I tried this and gave me a graphQL error.

Reply:
I tried this in the GraphiQL explorer at https://shopify.dev/custom-storefronts/tools/graphiql-storefront-api and it does definitely work. Can you please test this again?

Proof:

<img width="1319" alt="Screenshot 2022-06-01 at 00 18 40" src="https://user-images.githubusercontent.com/841956/171299137-fdf36177-4f6f-430d-98ac-73aa41b67067.png">

Reply:
Passing after an string of null worked. Empty string did not work.